src/NavBar.module.css

```diff
@@ -0,0 +1,67 @@
+.nav {
+  height: 100%;
+  background: var(--colors-bgPrimary);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: calc(var(--spacing-md) + .2rem);
+  padding-right: calc(var(--spacing-md) + .2rem);
```

Comment:
Increased from spacing-sm to allocate space for the tooltip
![Screenshot 2025-03-18 at 9 29 02 AM](https://github.com/user-attachments/assets/47f02039-c8be-454f-a2e2-544b1f1b2ec4)